frontend/urls.py

```diff
@@ -6,5 +6,6 @@
     path('', views.homepage),
     path('settings/', views.settings),
     path('verify/', views.verify),
-    path('twitch/', views.twitch)
+    path('twitch/', views.twitch),
+    path('socialAlert/', views.socialAlert)

```

Comment:
```suggestion
    path('socialalert/', views.socialAlert)

```

It doesn't look right in the webpage with the URL being camel-cased IMO